src/jobs/pricewatch-single.js

```diff
@@ -1,13 +1,16 @@
 const priceWatchSchema = require('../schemas/price-watch')
 const jobSchema = require('../schemas/jobs')
 const { getDexPrice } = require('../utils/external')
-const { timeNow } = require('../utils/helper')
+const { timeNow, getISODate } = require('../utils/helper')
+const { getExcludedGuilds } = require('../utils/jobs')
+const { DEFAULT_PRICE_WATCH_SINGLE_INTERVAL } = require('../constants/constants')
 
 module.exports = ({
   meta: {
     name: 'price-watch-single',
     description: 'Sends a single message of the current price of SafeMoon',
-    interval: 15 * 1000,
+    interval: 5 * 1000,
```

Comment:
This `interval` should never be less than the `cacheTime` variable, otherwise it'll just output the same data each time, and preferably never below 15 to avoid API abuse.

Reply:
I decreased it on purpose, so the difference between the passed time of certain guilds is not that high.

Let's say one guild has the time interval set to 15 seconds. The job run just one second before the interval passed for this guild. If that happens, this guild has to wait additional 15 seconds, before it will have the job executed. That makes 29 seconds for the job to be executed for that guild. However, if we set it just to 5 seconds. That guild would just have to wait, it would have to wait only additional 6 seconds, for the job to be executed.

The default interval is responsible here for not allowing to post the same data. So if the defaultInterval is set to 15 seconds and the user is not allowed to own interval below that value, the guild should not have the values repeated.